attestation-service/attestation-service/src/verifier/snp/mod.rs

```diff
@@ -148,36 +149,36 @@ pub(crate) fn verify_report_signature(
     Ok(())
 }
 
-pub fn load_milan_cert_chain() -> Result<(x509::X509, x509::X509)> {
-    let certs = x509::X509::stack_from_pem(include_bytes!("milan_ask_ark.pem"))?;
+fn load_milan_cert_chain() -> (X509, X509) {
+    let certs =
+        X509::stack_from_pem(include_bytes!("milan_ask_ark.pem")).expect("Malformed Milan ASK/ARK");
     if certs.len() != 2 {
-        bail!("Malformed Milan ASK/ARK");
+        panic!("Malformed Milan ASK/ARK");
```

Comment:
I am a little worried about `panic!` and `expect()` here. As attestation-service might be a service that serves different attestation requests. panic here would leave no logs to the upper caller. wdyt?

Reply:
hmm, ideally this wouldn't even compile (I'd make this const if that would be feasible). We're bundling vendor certificates in this verifier. If we fail to load them then something is seriously off. But I understand the concern that it might affect other code, if for some reason under some unforeseen circumstances this fails.

However, I think the right place to bail, if we fail to load those certs, is at verifier initialization. WDYT?

https://github.com/confidential-containers/kbs/blob/54820fd0d8b31725dbbd6cf8b9e50fa85ba7ea93/attestation-service/attestation-service/src/verifier/mod.rs#L50

Reply:
Sounds good to me!

Reply:
pushed. PTAL